src/game/behaviors/water_bomb.inc.c

```diff
@@ -64,25 +62,25 @@ void bhv_water_bomb_spawner_update(void) {
     }
 }
 
+static struct SpawnParticlesInfo sWaterBombExplodeParticles = {
+    /* behParam:        */ 0,
+    /* count:           */ 5,
+    /* model:           */ MODEL_BUBBLE,
+    /* offsetY:         */ 20,
+    /* forwardVelBase:  */ 20,
+    /* forwardVelRange: */ 60,
+    /* velYBase:        */ 10,
+    /* velYRange:       */ 10,
+    /* gravity:         */ -2,
+    /* dragStrength:    */ 0,
+    /* sizeBase:        */ 35.0f,
+    /* sizeRange:       */ 10.0f,
+};
+
 /**
  * Spawn particles when the water bomb explodes.
  */
 void water_bomb_spawn_explode_particles(s8 offsetY, s8 forwardVelRange, s8 velYBase) {
-    static struct SpawnParticlesInfo sWaterBombExplodeParticles = {
-        /* behParam:        */ 0,
-        /* count:           */ 5,
-        /* model:           */ MODEL_BUBBLE,
-        /* offsetY:         */ 20,
-        /* forwardVelBase:  */ 20,
-        /* forwardVelRange: */ 60,
-        /* velYBase:        */ 10,
-        /* velYRange:       */ 10,
-        /* gravity:         */ -2,
-        /* dragStrength:    */ 0,
-        /* sizeBase:        */ 35.0f,
-        /* sizeRange:       */ 10.0f,
-    };
-
```

Comment:
hmmm i don't think making this globally scoped is better practice, why not just keep the variable where its used?